src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs

```diff
@@ -363,13 +363,21 @@ static IReadOnlyCollection<Assembly> LoadConfigurationAssemblies(IConfigurationS
             {
                 if (string.IsNullOrWhiteSpace(simpleName))
                     throw new InvalidOperationException(
-                        "A zero-length or whitespace assembly name was supplied to a Serilog.Using configuration statement.");
+                        $"A zero-length or whitespace assembly name was supplied to a {usingSection.Path} configuration statement.");
 
                 var assembly = Assembly.Load(new AssemblyName(simpleName));
                 if (!assemblies.ContainsKey(assembly.FullName))
                     assemblies.Add(assembly.FullName, assembly);
             }
         }
+        else if (!assemblyFinder.CanFindAssemblies)
+        {
+            var message = $"The application is published as single-file and no {usingSection.Path} configuration section is defined.{Environment.NewLine}" +
+                          $"Either add a {usingSection.Path} section or explicitly specify assemblies that contains sinks and other types through the reader options. For example:{Environment.NewLine}" +
```

Comment:
can we raw string this

Reply:
Done in 631c61fbe10403fa435845f93ac5f68e35e80930 (force-pushed on top of the `dev` branch).